src/main/kotlin/com/terraformation/backend/accelerator/PreScreenVariableValuesFetcher.kt

```diff
@@ -0,0 +1,103 @@
+package com.terraformation.backend.accelerator
+
+import com.terraformation.backend.accelerator.model.PreScreenProjectType
+import com.terraformation.backend.accelerator.model.PreScreenVariableValues
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.accelerator.DeliverableId
+import com.terraformation.backend.db.default_schema.LandUseModelType
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.docprod.VariableId
+import com.terraformation.backend.documentproducer.db.VariableStore
+import com.terraformation.backend.documentproducer.db.VariableValueStore
+import com.terraformation.backend.documentproducer.model.ExistingNumberValue
+import com.terraformation.backend.documentproducer.model.ExistingSelectValue
+import com.terraformation.backend.documentproducer.model.ExistingValue
+import com.terraformation.backend.documentproducer.model.SelectVariable
+import com.terraformation.backend.documentproducer.model.Variable
+import com.terraformation.backend.log.perClassLogger
+import jakarta.inject.Named
+import java.math.BigDecimal
+
+@Named
+class PreScreenVariableValuesFetcher(
+    private val variableStore: VariableStore,
+    private val variableValueStore: VariableValueStore,
+) {
+  companion object {
+    const val STABLE_ID_NUM_SPECIES = "22"
+    const val STABLE_ID_PROJECT_TYPE = "3"
```

Comment:
Are these going to be inserted through a migration, or are these going to still be managed in the spreadsheet?

Reply:
Spreadsheet. I'd been considering a migration in part so we could reference the variables from a variety of tests, but then I realized it was better to encapsulate the variables this way such that we only need to care about the IDs in one place.